### PR TITLE
Feature/availability slot component new design #202

### PR DESCRIPTION
**Pull request by va-dym-ko-val**
image
image

### CODE REVIEW

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass [hunk 1]
```diff
@@ -1,16 +1,17 @@
 @import '../../../../../styles/color-variables'
 @import '../../../../../styles/screen-variables'
```
Comment (PublicFedorenko) [inline frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass]:
```suggestion
@import 'src/styles/color-variables'
@import 'src/styles/screen-variables'
```

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass [hunk 2]
```diff
@@ -31,22 +32,36 @@ app-side-menu-tabs
...
+            text-decoration: underline
+        .new-availability-header-label
+            font-size: 20px
+            font-weight: bold
+            color: $dark-gray
+            @media screen and (max-width: $tablet-min-width)
+                font-size: 16px
     mat-slide-toggle
-            margin-right: 10px
+                margin-right: 10px
```
Comment (PublicFedorenko) [inline frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass]: Why so many tabs here?

File: frontend/src/app/modules/header/header-item/header-item.component.sass [hunk 3]
```diff
@@ -121,7 +121,6 @@ label:hover
     .menu-team-btn
         label
             display: none
-
```
Comment (PublicFedorenko) [inline frontend/src/app/modules/header/header-item/header-item.component.sass]: Is it intended line removal?

**Review by PublicFedorenko — CHANGES_REQUESTED** [review]
The border around the General tab goes beyond the container. I would suggest to add more padding top.

There is white line when you hover over tabs. I would suggest to remove margin in parent container
image

**Review by PublicFedorenko — APPROVED** [review]

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass [hunk 4]
```diff
@@ -1,16 +1,17 @@
-@import '../../../../../styles/color-variables'
-@import '../../../../../styles/screen-variables'
+@import 'src/styles/color-variables'
+@import 'src/styles/screen-variables'
+@import url('https://rsms.me/inter/inter.css')
```
Comment (tatianahutii) [inline frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass]: why did you import this here?

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass (hunk 4 above)
Comment (va-dym-ko-val) [reply frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass]: It works only with font added locally in sass file

Comment (va-dym-ko-val) [reply frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.sass]: Without it
image

**Review by tatianahutii — APPROVED** [review]

**Review by apofis96 — APPROVED** [review]

**Pull request merged**